### ISSUE TITLE
Broken

### ISSUE THREAD

**Issue by hhsnopek** [carrot/10k-stopwatch#3]
Since the latest update on 1000ft, it has broke our stopwatch.

**Comment by kylemac** [comment]
funny enough, someone emailed us about that today as well. i don't know when we'll have time to diagnose and fix this

**Comment by hhsnopek** [comment]
That's how I discovered it hah - I'm looking into it now, hoping it's a quick fix

**Comment by BrandonRomano** [comment]
There was a complete overhaul of the UI -- I have a feeling this won't be a quick fix @hhsnopek, but definitely let us know if you find out how much effort it will be!

**Comment by hhsnopek** [comment]
This looks like a weekend issue, in short we need to:
- [ ] Update parent when appending the stopwatch `.stopwatch`
- [ ] Add & Update onClick listeners for disabling the timer
- [ ] Fix styling of stopwatch

**Comment by BrandonRomano** [comment]
That's not as bad as I thought

**Comment by hhsnopek** [comment]
Yeah I think the worst part is the latter, we will have to add some new listeners because of the drop down addition's (they trigger more `DOMNodeInserted`)

**Comment by BrandonRomano** [comment]
Oh yeah, I totally forgot this was a total hack to get working with that.

**Comment by hhsnopek** [comment]
Also the styling for the stopwatch is messed up currently (updated the list above)

**Comment by hhsnopek** [comment]
looking at this now, it's totally broken - basically just needs to be rewritten :disappointed:
